src/components/Card.js

```diff
@@ -13,22 +22,29 @@ class Card {
 
     likeButton.addEventListener("click", () => this._handleLikeIcon());
 
-    deleteButton.addEventListener("click", this._handleDelete.bind(this));
+    deleteButton.addEventListener("click", () => this._handleDelete(this));
 
     cardImage.addEventListener("click", () => {
       this._handleCardClick({ name: this._name, link: this._link });
     });
   }
 
+  updateLikesCount(likes) {
+    this._likes = likes;
+    const likesCountElement = this._element.querySelector(".card__likes-count");
+    likesCountElement.textContent = this._likes;
+  }
+
   _handleLikeIcon() {
     this._element
       .querySelector(".card__like-button")
       .classList.toggle("card__like-button_active");
   }
 
-  _handleDelete() {
-    this._element.remove();
-    this._element = null;
+  _handleDelete(card) {
+    this._handleDelete(card);
+    // this._element.remove();
```

Comment:
You should create a method `deleteCard`, which will contain the code for removing the card element and use it in index.js after removing the card on the server